Hunting Queries/SecurityEvent/FirewallRuleChanges_using_netsh.yaml

```diff
@@ -0,0 +1,30 @@
+id: 3dc5dc8b-160b-407e-9925-24a91e3599df
+name: Firewall rule changes using netsh
+description: |
+  This query will show firewall rule changes using netsh utility.
+  This technique was seen in relation to Solarigate attack but the results can indicate potential  malicious activity used in different attacks.
+  Refer to netsh syntax: https://docs.microsoft.com/windows-server/administration/windows-commands/netsh
```

Comment:
Include a reference to our M365D blog - https://www.microsoft.com/security/blog/2021/01/20/deep-dive-into-the-solorigate-second-stage-activation-from-sunburst-to-teardrop-and-raindrop/